dbtemplates/admin.py

```diff
@@ -2,7 +2,13 @@
 from django import forms
 from django.contrib import admin
 from django.core.exceptions import ImproperlyConfigured
-from django.utils.translation import ungettext, ugettext_lazy as _
+try:
+    #Django 3 and bellow
+    from django.utils.translation import ugettext_lazy, ngettext as _
+except ImportError:
+    #Django 4+
+    from django.utils.translation import ngettext, gettext_lazy as _
```

Comment:
This is actually a bit wrong. Original import wants ugettext_lazy and ngettext as _. Your import imports ngettext as ngettext and gettext_lazy as _. This makes tests fail. Please correct.

Reply:
Sorry, missed this comment until now.